src/utils/gauges.ts

```diff
@@ -98,6 +100,14 @@ export async function getGaugeData(
       )
     ).map((poolAddress) => poolAddress.toLowerCase())
 
+    const gaugePoolNames: string[] = (
```

Comment:
I kind of have mixed feelings about this... 

On one hand, having the poolName on a gauge would be convenient for some things.

On the other hand, `gaugeToPoolData` returns a bunch of stuff, is probably kind of slow, and we already have all that data in `BasicPools`. We can get all the info we need about a gauge's pool by doing something like this in the TokenClaimDialog
```
basicPools.map(pool => {
  const gauge = gaugeData.gauges[pool.poolAddress]
  if (!gauge) return null
  // do gauge stuff with full pool info
}
```

With that said, it would also be nice to be able to look up a pool by its address instead of just its name. P1.